docs/browser/events.md

```diff
@@ -0,0 +1,158 @@
+# Semantic conventions for browser events
+
+**Status**: [Experimental](../../../document-status.md)
+
+<details>
+<summary>Table of Contents</summary>
+<!-- Re-generate TOC with `markdown-toc --no-first-h1 -i` -->
+
+<!-- toc -->
+
+- [PageView](#pageview)
+- [PageNavigationTiming](#pagenavigationtiming)
+- [ResourceTiming](#resourcetiming)
+- [Exception](#exception)
+- [UserAction](#useraction)
+- [WebVital](#webvital)
+
+<!-- tocstop -->
+
+</details>
+
+This document describes the semantic conventions for events on browser platform. All browser events MUST use a namespace of `browser` in the `event.name` property.
+
+## PageView
+
+The event name MUST be `browser.page_view`.
+
+This event describes the details of the web page visited.
```

Comment:
**Question:** Would it make sense to outline what the trace & span ID represent in case a physical page is loaded? For example, does it represent a trace started on the client or would it represent the trace through which the HTML document was generated?

Reply:
I believe this is a question for the instrumentation that will emit this event. We are only describing the payload structure of this event without saying anything about the context in which the event is emitted.